docs/install/regression.md

```diff
@@ -0,0 +1,34 @@
+# Regression
```

Comment:
This file is not required - the contents not valid now.

Reply:
Should this file be removed?